dali/server/daserver.cpp

```diff
@@ -462,74 +462,70 @@ int main(int argc, const char* argv[])
         serverConfig.setown(loadConfiguration(defaultYaml, argv, "dali", "DALI", DALICONF, nullptr));
         Owned<IFile> sentinelFile = createSentinelTarget();
         removeSentinelFile(sentinelFile);
-        port = serverConfig->getPropInt("@port");
-#ifndef _CONTAINERIZED
-        if (!checkCreateDaemon(argc, argv))
-            return EXIT_FAILURE;
-
-        for (unsigned i=1;i<(unsigned)argc;i++) {
-            if (streq(argv[i],"--daemon") || streq(argv[i],"-d")) {
-                i++; // consumed within checkCreateDaemon(), bump up here
-            }
-            else if (streq(argv[i],"--server") || streq(argv[i],"-s"))
-                server = argv[++i];
-            else if (streq(argv[i],"--port") || streq(argv[i],"-p"))
-                port = atoi(argv[++i]);
-            else if (streq(argv[i],"--rank") || streq(argv[i],"-r"))
-                myrank = atoi(argv[++i]);
-            else if (!startsWith(argv[i],"--config"))
-            {
-                usage();
-                return EXIT_FAILURE;
-            }
+        StringBuffer dataPath;
+        StringBuffer mirrorPath;
+        if (isContainerized())
+        {
+            port = serverConfig->getPropInt("service/@port");
```

Comment:
NB: this is the actual fix - in containerized pick up @port from service/, vs non-containerized (as it was) on line 481.

The rest of the code differences are moving code into a if (isContainerized()) structure.